GI03FLMA/GI03FLMA_R00005_ingest.csv

```diff
@@ -0,0 +1,42 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00005/instruments/CTDMO_sn37-12221/*.hex,GI03FLMA-RI000-00-CTDMOH000,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00005/instruments/CTDMO_sn37-12226/*.hex,GI03FLMA-RI000-00-CTDMOH100,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00005/instruments/CTDMO_sn37-12223/*.hex,GI03FLMA-RI000-00-CTDMOH400,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00005/instruments/CTDMO_sn37-12222/*.hex,GI03FLMA-RI000-00-CTDMOH700,recovered_inst,Available,
+mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00005/instruments/VELPT_snAQD-11973/*.aqd,GI03FLMA-RI000-00-VELPTB000,recovered_inst,Available,
+mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00005/instruments/VELPT_snAQD-12233/*.aqd,GI03FLMA-RI000-00-VELPTB100,recovered_inst,Available,
+mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00005/instruments/VELPT_snAQD-11976/*.aqd,GI03FLMA-RI000-00-VELPTB400,recovered_inst,Available,
+mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00005/instruments/VELPT_snAQD-11974/*.aqd,GI03FLMA-RI000-00-VELPTB700,recovered_inst,Available,
+mi.dataset.driver.sio_eng.sio.sio_eng_sio_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00005/cg_data/MSIOC_sn*/STATUS/STA*.DAT,GI03FLMA-RIM01-00-SIOENG000,recovered_host,Available,
+mi.dataset.driver.adcps_jln.sio.adcps_jln_sio_telemetered_driver,/omc_data/whoi/OMC/GI03FLMA/R00005/cg_data/MSIOC_sn1488303/ADCPS/Raw/ADCPS/*.DAT,GI03FLMA-RIM01-02-ADCPSL003,recovered_host,Available,

```

Comment:
Wondering why you used the MSIOC S/N here instead of an *.

Reply:
No idea. I think I was just verifying the MSIOC persistor S/N to myself. Agree MSIOC_sn* is better. Will update